stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequestV2.kt

```diff
@@ -126,9 +128,9 @@ class AnalyticsRequestV2(
     /**
      * Parameters required by r.stripe.com
      */
-    private fun analyticParams() = mapOf(
+    private fun analyticParams(): Map<String, Any> = mapOf(
         PARAM_CLIENT_ID to clientId,
-        PARAM_CREATED to System.currentTimeMillis() / MILLIS_IN_SECOND,
+        PARAM_CREATED to System.currentTimeMillis().milliseconds.toDouble(SECONDS),
```

Comment:
we can also divide by `1000.0`, but the Kotlin duration API might make this more readable.